### ISSUE TITLE
Improve answer "common/default settings"

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#77]
Die Antwort "common/default settings already available (e.g. in /etc/papersize) " zur Frage "Configuration integration is an effort to adapt applications better to the system. Why did you or would you introduce configuration integration? (e.g. reading /etc/papersize)" fand ich ein wenig verwirrend. Impliziert das die Antwort "I would not"?

**Comment by Namoshek** [comment]
Maybe add "because" (and verb; also because of consistency with other answers for this question):
> because common/default settings are already available (e.g. in /etc/papersize)

We could also rephrase it to give a hint that "shared/system-wide configurations" are meant by this answer...

**Comment by markus2330** [comment]
Yes, maybe only the inconsistency was causing trouble

**Comment by omnidan** [comment]
I updated the answer to `because common/default settings are already available (e.g. in /etc/papersize)`

**Comment by sirblackheart** [comment]
nice

**Comment by markus2330** [comment]
Thank you, nice job!
